src/lib/gas-refund/gas-refund-api.ts

```diff
@@ -165,19 +180,35 @@ export class GasRefundApi {
     };
   }
 
-  async _fetchMerkleData(address: string): Promise<GasRefundClaim[]> {
-    const grpDataResult: GasRefundClaim[] = await Database.sequelize.query(
-      MERKLE_DATA_SQL_QUERY,
-      {
+  async _fetchMerkleData(
+    address: string,
+    network: number,
+  ): Promise<AddressMerkleData> {
+    const gasRefundParticipationsOptions: { where: any } = {
+      where: {
+        address,
+        epoch: {
+          [Op.gte]: EPOCH_WHEN_MCS_ENABLED,
```

Comment:
I don't quite get the changes in `src/lib/gas-refund/gas-refund-api.ts`

both queries only return distribgutions for epochs greater than 37.
In the result legacy stuff is not fetchable and ultimately the endpoint is not backwards-compatible.

I'm going to revert changes in this file and in the `router.ts`.

I think this endpoint should continue to work as it was -- return merkle proof, claim status and amount info for the participations by user, so that this data can be used for claiming.

I'm assuming the intent behind changing these two files was to serve breakdown data. 

If it's so, then I've added another API route to [solve specifically that](https://github.com/paraswap/paraswap-volume-tracker/pull/93/files#diff-bb8ff0dd16ff286e4086240831f70dd9233de8896f2f50219eeff29414affe64R239)

Let me know if I'm missing something